config/bad/bad-dup-names.conf

```diff
@@ -0,0 +1,98 @@
+
+server
+{
+	main
+	{
+		host 127.0.0.1
+		listen 8000
+		server_name localhost
```

Comment:
we should use some other name than the localhost, its a convention and reserved by default already to the ip 127.0.0.1 ...

Reply:
that's fine. i tried a bunch of options, and that was the last one. i'll update it later